WooCommerce/Classes/ViewRelated/Filters/FilterListViewController.swift

```diff
@@ -206,8 +201,8 @@ private extension FilterListViewController {
     }
 
     func observeListSelectorCommandItemSelection() {
-        cancellableSelectedFilterType?.cancel()
-        cancellableSelectedFilterType = listSelectorCommand.onItemSelected.subscribe { [weak self] selected in
+        selectedFilterTypeSubscription?.cancel()
+        selectedFilterTypeSubscription = listSelectorCommand.onItemSelected.sink { [weak self] selected in
```

Comment:
I think the old subscription should be canceled when we assign a new reference to it, so the `cancel()` method is redundant here. WDYT?

Reply:
yea I don't think we need to cancel with Combine now! Updated in https://github.com/woocommerce/woocommerce-ios/pull/5989/commits/97e9eba8ec2623c15ffaf10dd7f746b401aa4cc7